database/migrations/2024_06_21_120622_create_comments_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('comments', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('user_id')->unsigned();
+            $table->integer('post_id')->unsigned();
+            $table->integer('parent_id')->unsigned()->nullable();
```

Comment:
Can we remove this column now?

Reply:
Ah. I think you've removed this field in a different migration. It would be a better (and cleaner) option to rollback your migrations and simply remove the column here, then re-migrate.

It'll mean this migration - database/migrations/2024_06_25_090448_alter_comments_table_fields.php - is not needed.